code/modules/projectiles/guns/magic/rune.dm

```diff
@@ -129,7 +129,7 @@
 
 /obj/item/gun/magic/rune/bullet_rune
 	name = "Bullet Rune"
-	desc = "This obsidian rune has the ability to shoot bullets out of it. I have yet to see one who will outsmart them"
+	desc = "This obsidian rune has the ability to shoot bullets out of it. I have yet to see someone who will outrun them."
```

Comment:
Ktl confirmed boomer

Reply:
Ktl not understanding classic video game references lol